frontend/src/scenes/web-analytics/WebDashboard.tsx

```diff
@@ -259,7 +259,7 @@ export const WebTabs = ({
                     )}
                 </h2>
 
-                {tabs.length > 4 ? (
+                {tabs.length > 5 ? (
```

Comment:
Keep as 4 to avoid breaking mobile layout, but let's come up with a better fix long term